src/charm.py

```diff
@@ -230,8 +236,7 @@ def _push_namespace_labels(self):
 
     def _on_kfam_pebble_ready(self, event):
         """Update the started kfam container."""
-        if not self.kfam_container.can_connect():
-            raise ErrorWithStatus("kfam Pebble is ready and container is not ready", BlockedStatus)
+        self._check_container_connection(self.kfam_container)
```

Comment:
charm will go into error state here

Reply:
That's what we are doing in other charms. Is there an issue with this particular charm?

Reply:
I see what you mean, changed.